### PR TITLE
Creating a more user-friendly interface

### PR DESCRIPTION
**Pull request by StanislavBeliaev**
Now when you create a new user, form will give you a messages if you did something wrong.Burger menu was added for easier site navigation.Logout button was added.

### CODE REVIEW

File: src/pages/MemberTasks.jsx [hunk 1]
```diff
@@ -44,6 +55,13 @@ function MembersTasks() {
 
     return (
         <div className={classes.MembersTasksContainer}>
+            <p>
+                Hi! Dear{' '}
+                {userData.map(([id, val], idx) => {
```
Comment (YauhenKoipish) [inline src/pages/MemberTasks.jsx:60]: I think here we can avoid using map

**Review by YauhenKoipish — CHANGES_REQUESTED** [review]

**Comment by StanislavBeliaev** [comment]
Added secure routes, burger menu is not available if you are not logged in

File: src/App/App.jsx [hunk 2]
```diff
@@ -9,28 +12,114 @@ import Tasks from 'pages/Tasks';
...
+    const [accToken, setAccToken] = useState({ token: '' });
+    const logOut = () => {
+        setIsLoggedin(false);
+        setAccToken({ token: '' });
+    };
+    const [burgerStatus, setBurgerStatus] = useState(false);
+    const burger = () => {
+        setBurgerStatus((prev) => !prev);
+    };
+    console.log(burgerStatus);
```
[the remark this answers is not in the document]
Comment (StanislavBeliaev) [reply src/App/App.jsx]: done

File: src/App/App.jsx [hunk 3]
```diff
@@ -9,28 +12,114 @@ import Tasks from 'pages/Tasks';
...
+                            fill='currentColor'
+                            className='bi bi-list'
+                            viewBox='0 0 16 16'
+                            style={{ color: 'white' }}
+                        >
+                            <path
+                                fillRule='evenodd'
+                                d='M2.5 12a.5.5 0 0 1 .5-.5h10a.5.5 0 0 1 0 1H3a.5.5 0 0 1-.5-.5zm0-4a.5.5 0 0 1 .5-.5h10a.5.5 0 0 1 0 1H3a.5.5 0 0 1-.5-.5zm0-4a.5.5 0 0 1 .5-.5h10a.5.5 0 0 1 0 1H3a.5.5 0 0 1-.5-.5z'
+                            />
+                        </svg>
```
[the remark this answers is not in the document]
Comment (StanislavBeliaev) [reply src/App/App.jsx]: done

File: src/App/App.jsx [hunk 4]
```diff
@@ -9,28 +12,114 @@ import Tasks from 'pages/Tasks';
...
+                                        style={{ color: '#d9534f' }}
+                                    >
+                                        <path
+                                            fillRule='evenodd'
+                                            d='M6 12.5a.5.5 0 0 0 .5.5h8a.5.5 0 0 0 .5-.5v-9a.5.5 0 0 0-.5-.5h-8a.5.5 0 0 0-.5.5v2a.5.5 0 0 1-1 0v-2A1.5 1.5 0 0 1 6.5 2h8A1.5 1.5 0 0 1 16 3.5v9a1.5 1.5 0 0 1-1.5 1.5h-8A1.5 1.5 0 0 1 5 12.5v-2a.5.5 0 0 1 1 0v2z'
+                                        />
+                                        <path
+                                            fillRule='evenodd'
+                                            d='M.146 8.354a.5.5 0 0 1 0-.708l3-3a.5.5 0 1 1 .708.708L1.707 7.5H10.5a.5.5 0 0 1 0 1H1.707l2.147 2.146a.5.5 0 0 1-.708.708l-3-3z'
+                                        />
```
[the remark this answers is not in the document]
Comment (StanislavBeliaev) [reply src/App/App.jsx]: done

File: src/App/App.jsx [hunk 5]
```diff
@@ -9,28 +12,114 @@ import Tasks from 'pages/Tasks';
...
+                                    >
+                                        <path
+                                            fillRule='evenodd'
+                                            d='M10 12.5a.5.5 0 0 1-.5.5h-8a.5.5 0 0 1-.5-.5v-9a.5.5 0 0 1 .5-.5h8a.5.5 0 0 1 .5.5v2a.5.5 0 0 0 1 0v-2A1.5 1.5 0 0 0 9.5 2h-8A1.5 1.5 0 0 0 0 3.5v9A1.5 1.5 0 0 0 1.5 14h8a1.5 1.5 0 0 0 1.5-1.5v-2a.5.5 0 0 0-1 0v2z'
+                                        />
+                                        <path
+                                            fillRule='evenodd'
+                                            d='M15.854 8.354a.5.5 0 0 0 0-.708l-3-3a.5.5 0 0 0-.708.708L14.293 7.5H5.5a.5.5 0 0 0 0 1h8.793l-2.147 2.146a.5.5 0 0 0 .708.708l3-3z'
+                                        />
+                                    </svg>
```
[the remark this answers is not in the document]
Comment (StanislavBeliaev) [reply src/App/App.jsx]: done

File: src/pages/LoginPage.jsx [hunk 6]
```diff
@@ -17,12 +20,11 @@ function LoginPage() {
...
         const email = target.email.value;
         const password = target.password.value;
-
         signInWithEmailAndPassword(auth, email, password)
             .then((userCredential) => {
                 // Signed in
                 setAccToken({ token: userCredential.user.accessToken });
-
+                setLogin(true);
                 console.log(userCredential);
```
[the remark this answers is not in the document]
Comment (StanislavBeliaev) [reply src/pages/LoginPage.jsx]: done

File: src/pages/LoginPage.jsx [hunk 7]
```diff
@@ -35,21 +37,97 @@ function LoginPage() {
...
+                                        className='bi bi-person-circle'
+                                        viewBox='0 0 16 16'
+                                        style={svgStyle}
+                                    >
+                                        <path d='M11 6a3 3 0 1 1-6 0 3 3 0 0 1 6 0z' />
+                                        <path
+                                            fillRule='evenodd'
+                                            d='M0 8a8 8 0 1 1 16 0A8 8 0 0 1 0 8zm8-7a7 7 0 0 0-5.468 11.37C3.242 11.226 4.805 10 8 10s4.757 1.225 5.468 2.37A7 7 0 0 0 8 1z'
+                                        />
+                                    </svg>
```
[the remark this answers is not in the document]
Comment (StanislavBeliaev) [reply src/pages/LoginPage.jsx]: done

File: src/pages/LoginPage.jsx [hunk 8]
```diff
@@ -35,21 +37,97 @@ function LoginPage() {
...
+                                    >
+                                        <path d='M5.338 1.59a61.44 61.44 0 0 0-2.837.856.481.481 0 0 0-.328.39c-.554 4.157.726 7.19 2.253 9.188a10.725 10.725 0 0 0 2.287 2.233c.346.244.652.42.893.533.12.057.218.095.293.118a.55.55 0 0 0 .101.025.615.615 0 0 0 .1-.025c.076-.023.174-.061.294-.118.24-.113.547-.29.893-.533a10.726 10.726 0 0 0 2.287-2.233c1.527-1.997 2.807-5.031 2.253-9.188a.48.48 0 0 0-.328-.39c-.651-.213-1.75-.56-2.837-.855C9.552 1.29 8.531 1.067 8 1.067c-.53 0-1.552.223-2.662.524zM5.072.56C6.157.265 7.31 0 8 0s1.843.265 2.928.56c1.11.3 2.229.655 2.887.87a1.54 1.54 0 0 1 1.044 1.262c.596 4.477-.787 7.795-2.465 9.99a11.775 11.775 0 0 1-2.517 2.453 7.159 7.159 0 0 1-1.048.625c-.28.132-.581.24-.829.24s-.548-.108-.829-.24a7.158 7.158 0 0 1-1.048-.625 11.777 11.777 0 0 1-2.517-2.453C1.928 10.487.545 7.169 1.141 2.692A1.54 1.54 0 0 1 2.185 1.43 62.456 62.456 0 0 1 5.072.56z' />
+                                        <path d='M9.5 6.5a1.5 1.5 0 0 1-1 1.415l.385 1.99a.5.5 0 0 1-.491.595h-.788a.5.5 0 0 1-.49-.595l.384-1.99a1.5 1.5 0 1 1 2-1.415z' />
+                                    </svg>
```
[the remark this answers is not in the document]
Comment (StanislavBeliaev) [reply src/pages/LoginPage.jsx]: done

File: src/pages/MemberTasks.jsx (hunk 1 above)
Comment (StanislavBeliaev) [reply src/pages/MemberTasks.jsx]: done

**Pull request merged**